### PR TITLE
Mkosi dependency naming adjustment

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

**Review by bluca — APPROVED** [review]
lgtm

File: .mkosi/mkosi.arch [hunk 1]
```diff
@@ -52,4 +52,3 @@ Packages=
         nano
         qrencode
         strace
-        vi
```
Comment (poettering) [inline .mkosi/mkosi.arch:55]: but but but, my finger memory, but but but.

**Comment by poettering** [comment]
lgtm

**Comment by bluca** [comment]
mkosi changes, no need to wait for bionic-*

**Pull request merged**